geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/fs/storage/parquet/io/SimpleFeatureReadSupport.scala

```diff
@@ -29,12 +30,15 @@ import scala.collection.mutable.ArrayBuffer
 
 class SimpleFeatureReadSupport extends ReadSupport[SimpleFeature] {
 
-  private var schema: SimpleFeatureParquetSchema = _
+  var schema: SimpleFeatureParquetSchema = null
+  var schemaVersion: Integer = null
```

Comment:
these should stay private if at all possible - we don't want to expose mutable state outside the scope of the class